tests/Integration/Controller/ListControllerTest.php

```diff
@@ -320,4 +331,83 @@ public function getListMembersWithCurrentSessionKeyForExistingListWithSubscriber
+     * @test
+     */
+    public function getListCountSubscribersForExistingListWithoutSessionKeyReturnsForbiddenStatus()
+    {
+        $this->getDataSet()->addTable(static::LISTS_TABLE_NAME, __DIR__ . '/Fixtures/SubscriberList.csv');
+        $this->applyDatabaseChanges();
+
+        $this->client->request('get', '/api/v2/lists/1/count/subscribers');
+
+        $this->assertHttpForbidden();
+    }
+
+    /**
+     * @test
+     */
+    public function getListCountSubscribersForExistingListWithExpiredSessionKeyReturnsForbiddenStatus()
+    {
+        $this->getDataSet()->addTable(static::LISTS_TABLE_NAME, __DIR__ . '/Fixtures/SubscriberList.csv');
+        $this->getDataSet()->addTable(static::ADMINISTRATOR_TABLE_NAME, __DIR__ . '/Fixtures/Administrator.csv');
+        $this->getDataSet()->addTable(static::TOKEN_TABLE_NAME, __DIR__ . '/Fixtures/AdministratorToken.csv');
+        $this->applyDatabaseChanges();
+
+        $this->client->request(
+            'get',
+            '/api/v2/lists/1/count/subscribers',
+            [],
+            [],
+            ['PHP_AUTH_USER' => 'unused', 'PHP_AUTH_PW' => 'cfdf64eecbbf336628b0f3071adba763']
+        );
+
+        $this->assertHttpForbidden();
+    }
+
+    /**
+     * @test
+     */
+    public function getListCountSubscribersWithCurrentSessionKeyForExistingListReturnsOkayStatus()
+    {
+        $this->getDataSet()->addTable(static::LISTS_TABLE_NAME, __DIR__ . '/Fixtures/SubscriberList.csv');
+        $this->applyDatabaseChanges();
+
+        $this->authenticatedJsonRequest('get', '/api/v2/lists/1/count/subscribers');
+
+        $this->assertHttpOkay();
+    }
+
+    /**
+     * @test
+     */
+    public function getListCountSubscribersWithCurrentSessionKeyForExistingListWithNoSubscribersReturnsZero()
+    {
+        $this->getDataSet()->addTable(static::LISTS_TABLE_NAME, __DIR__ . '/Fixtures/SubscriberList.csv');
+        $this->getDataSet()->addTable(static::SUBSCRIBER_TABLE_NAME, __DIR__ . '/Fixtures/Subscriber.csv');
+        $this->getDataSet()->addTable(static::SUBSCRIPTION_TABLE_NAME, __DIR__ . '/Fixtures/Subscription.csv');
+        $this->applyDatabaseChanges();
+
+        $this->authenticatedJsonRequest('get', '/api/v2/lists/3/count/subscribers');
+        $responseContent = $this->getResponseContentAsInt();
+
+        static::assertSame(0, $responseContent);
+    }
+
+    /**
+     * @test
+     */
+    public function getListCountSubscribersWithCurrentSessionKeyForExistingListWithSubscribersReturnsSubscribersCount()
+    {
+        $this->getDataSet()->addTable(static::LISTS_TABLE_NAME, __DIR__ . '/Fixtures/SubscriberList.csv');
+        $this->getDataSet()->addTable(static::SUBSCRIBER_TABLE_NAME, __DIR__ . '/Fixtures/Subscriber.csv');
+        $this->getDataSet()->addTable(static::SUBSCRIPTION_TABLE_NAME, __DIR__ . '/Fixtures/Subscription.csv');
+        $this->applyDatabaseChanges();
+
+        $this->authenticatedJsonRequest('get', '/api/v2/lists/2/count/subscribers');
```

Comment:
As discussed, this should be `/lists/2/subscribers/count`, not `/api/v2/lists/2/count/subscribers`.

Reply:
OK, I misread your comment.